tests/test_matrix_utils.py

```
@@ -6222,6 +6297,22 @@ def test_iter_room_alias_entries_list_format(self, mock_logger):
         entries[0][1]("!newroom:example.com")
         self.assertEqual(mapping[0], "!newroom:example.com")
 
+    @patch("mmrelay.matrix_utils.logger")
+    def test_iter_room_alias_entries_dict_format(self, mock_logger):
```

Ruff (0.14.10)

Unused method argument: `mock_logger`

(ARG002)